src/main/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/InterchangeHeader.java

```diff
@@ -26,7 +26,7 @@
 @AllArgsConstructor
 public class InterchangeHeader extends Segment {
 
-    public static final String KEY = "UNB";
+    protected static final String KEY = "UNB";
```

Comment:
Is there a reason why we've got KEY as `protected` instead of using the `getKey` Getter method?

Reply:
It's a constant class variable - wouldn't normally expose them with a method.